package.json

```diff
@@ -22,7 +22,6 @@
     "node-sass": "4.11.0",
```

Comment:
Please remove `node-sass` & `gatsby-plugin-sass` as well.
I think we don't need it anymore since we're using Ant Design which uses LESS instead of SASS.

Reply:
I've been remove `node-sass`